### ISSUE TITLE
Copy and paste

### ISSUE THREAD

**Issue by talesam** [jlesage/docker-firefox#168]
How do I copy something from my machine to the FF container and from FF to my machine?

Is it possible to do something like VirtualBox does, let the host and client communicate, copying something to each other as if it were native?

**Comment by jlesage** [comment]
The control menu has a clipboard that can be used for this purpose.

**Comment by jlesage** [comment]
What is greyed out ?  Once you type something under `Clipboard`, you should be able to past it (ctrl+v) into Firefox.

**Comment by 0xMH** [comment]
I can't find this menu. Can someone explain how to open it?

**Comment by jlesage** [comment]
@0xMH, this menu is automatically displayed when you load the page.  It then automatically closes after a few seconds.  You can re-open it by clicking the small handle.

**Comment by seoetraffic** [comment]
I have the same proplem, I cannot copy from firefox container to my local computer

**Comment by linux-y** [comment]
I have the same proplem, I cannot copy from firefox container to my local computer

**Comment by jlesage** [comment]
Are you using the clipboard from the control menu?

**Comment by kidtaroi** [comment]
Unable to coopy from the firefox container to local computer.

**Comment by jlesage** [comment]
Please provide more details. Are you using the clipboard box from the side panel?

**Comment by kidtaroi** [comment]
Yes, clipboard works when copying from local computer to firefox container but not vice versa? Is there a feature where we can natively copy ? between the container and the client machine ?

**Comment by kidtaroi** [comment]
I restarted the container and the clipboard copy paste worked both ways.  Can we request a native clipboard feature so that we can copy directly from/to the container?

**Comment by jlesage** [comment]
Note that currently, only Chrome offer the required APIs to perform "seamless" copy/paste functionality. Other browsers, like Firefox and Safari, require explicit user action.

**Comment by toozande** [comment]
how can i copy paste file ?

**Comment by jlesage** [comment]
To exchange files between the host and the container, you can use the folder you mapped to `/config`.

**Comment by coolspot18** [comment]
> What is greyed out ?  Once you type something under `Clipboard`, you should be able to past it (ctrl+v) into Firefox.

How do you paste on mobile like iOS where CTRL+V may not be available on the keyboard? Is there a button?

**Comment by jlesage** [comment]
You can right-click (2-fingers tap) and then use the Paste option from the menu.
